tests/Pest.php

```diff
@@ -17,6 +17,7 @@
 
 uses(Tests\TestCase::class)
     ->beforeEach(fn () => resolve(name: Filesystem::class)->deleteDirectory(basePath()))
+    ->afterEach(fn () => resolve(name: Filesystem::class)->deleteDirectory(base_path(path: applicationPath() . '/Http/User')))
```

Comment:
what http/user? shouldn't be like that i guess

Reply:
because the result of `application:make:request StoreUserRequest User` is on `app/Http/User/Request/StoreUserRequest.php` so that for remove the result

Reply:
hmmm as a deal before i thought we deal with App/Http/Request/User?